pkg/cmd/list/list.go

```diff
@@ -0,0 +1,95 @@
+// Lists the latests builds for a project.
+package list
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bufbuild/connect-go"
+	"github.com/depot/cli/pkg/api"
+	"github.com/depot/cli/pkg/config"
+	"github.com/depot/cli/pkg/project"
+	cliv1 "github.com/depot/cli/pkg/proto/depot/cli/v1"
+	"github.com/docker/cli/cli"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdLast() *cobra.Command {
+	var projectID string
+	var token string
+
+	cmd := &cobra.Command{
+		Use:     "list",
```

Comment:
Maybe you guys already chatted about this, but should we put this in a subcommand like `depot builds list`?

Reply:
> Maybe you guys already chatted about this, but should we put this in a subcommand like `depot builds list`?

Right on.  I like the subcommand idea a lot better.